src/reducers/viewers/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Reducer, Action } from 'redux';
+
+export interface ViewersState {
+    viewers: number;
+    change: number;
+}
+
+export interface UpdateViewersAction {type: "Update Viewers",
+    viewers: ViewersState["viewers"],
+    change: ViewersState["change"]}
```

Comment:
Change isn't necessary. The Reducer sees the last version of the state, so it can calculate the difference.